wapitiCore/attack/mod_http_headers.py

```diff
@@ -0,0 +1,78 @@
+import requests
+from wapitiCore.attack.attack import Attack
+from wapitiCore.net.web import Request
+from wapitiCore.language.vulnerability import Additional
+#This module check the recommendations of security parameters in HTTP headers
+class mod_http_headers(Attack):
+
+    name = "http_headers"
+    check_list = ['DENY', 'SAMEORIGIN', 'ALLOW-FROM']
+
+    def is_set(self, request: object, header_name, check_list):
```

Comment:
Parameter should be called `response` instead of `request` ;-)

Reply:
Fixed